workflow_scripts/test_utils.py

```diff
@@ -49,11 +54,14 @@ def get_model_and_test_data(directory_path):
     return onnx_model, test_data_set
 
 
+def remove_dir(directory_path):
+    if os.path.exists(directory_path) and os.path.isdir(directory_path):
+        shutil.rmtree(directory_path)
+
+
 def remove_tar_dir():
```

Comment:
I'd just replace these functions entirely with the one line of code that is inside the function.

Functions like this are not adding any value because they don't encapsulate anything useful.